lib/eventboss/development_mode.rb

```diff
@@ -17,6 +17,9 @@ def setup_infrastructure(queues)
           logger.info('development-mode') { "Creating queue #{queue.name}..." }
           sqs_client.create_queue(queue_name: queue.name)
 
+          logger.info('development-mode') { "Creating deadletter queue #{queue.name}-deadletter..." }
```

Comment:
Do we need these actually?

Reply:
We need this to test for example rake task in development mode. But if its only needed for my testing - I can remove it.